packages/gestalt/src/Contents.js

```diff
@@ -498,6 +498,10 @@ export default class Contents extends React.Component<Props, State> {
     const visibility = mainDir === null ? 'hidden' : 'visible';
     const background = `${bgColor}Bg`;
     const stroke = bgColor === 'white' ? '#ddd' : null;
+    const minDims =
```

Comment:
@NEsfandiari an alternative would be to remove the `minDimensions` when the width is a number